tdrs-backend/tdpservice/parsers/schema_defs/__init__.py

```diff
@@ -1,5 +1,7 @@
 from .header import header
 from .trailer import trailer
+from . import tanf
 
 header = header
 trailer = trailer
+tanf = tanf
```

Comment:
Why do we need these assignments?

Reply:
Nit: I would vote against it and would remove them and go with the second approach for importing.